src/glmfit.jl

```diff
@@ -507,13 +507,25 @@ function dispersion(m::AbstractGLM, sqr::Bool=false)
 end
 
 """
-    predict(mm::AbstractGLM, newX::AbstractMatrix; offset::FPVector=Vector{eltype(newX)}(0))
-
-Form the predicted response of model `mm` from covariate values `newX` and, optionally,
-an offset.
+    predict(mm::AbstractGLM, newX::AbstractMatrix; offset::FPVector=eltype(newX)[],
+            interval::Union{Symbol,Nothing}=nothing, level::Real = 0.95,
+            interval_method::Symbol = :transformation)
+
+If `interval` is `nothing` (the default), return the predicted response of model
+`mm` from covariate values `newX` and, optionally, an `offset`. If `interval` is
+`:confidence`, also return upper and lower bounds for a given coverage `level`. By default
+(`interval_method = :transformation`) the intervals are constructed by applying the
+inverse link to intervals for the linear predictor. If `interval_method = :delta`,
```

Comment:
```suggestion
```suggestion
Return the predicted response of model `mm` from covariate values `newX` and,
optionally, an `offset`.

If `interval=:confidence`, also return upper and lower bounds for a given coverage `level`.
By default (`interval_method = :transformation`) the intervals are constructed by applying
the inverse link to intervals for the linear predictor. If `interval_method = :delta`,
```